template.go

```diff
@@ -75,16 +87,15 @@ func (r *{{$typename}}) Scan(i interface{}) error {
 	case []byte:
 		return r.UnmarshalJSON(t)
```

Comment:
@quintenpalmer Does the type we get from gorp depend on the db column type or the value returned from the value() function? If the first, we still need this clause and should do as you suggested. If not, we should be able to remove this case entirely.